### DISCUSSION TITLE
Windows MonaServer2.exe for Non Developer ?

### DISCUSSION THREAD

**Discussion by bolli20000** [MonaSolutions/MonaServer2#64] [General]
Hi MonaServer2 Gurus, 

SOS - I have no developers PC to build the MonaServer2.exe for trial on Windows platform. Could somebody provide me a Windows 64 Bit .exe. of the current MonaServer2, please ? And sent it to my E-Mail: saboteur@gmx.de ? 

Thank you very much in advance. 

Best regards, Thomas (saboteur@gmx.de)

**Comment by thomasjammet** [comment]
Hi Thomas,

Please use the sourceforge link : https://sourceforge.net/projects/monaserver/files/MonaTiny/MonaTiny_Win64.zip/download

Regards

Thomas

**Comment by thomasjammet** [comment]
Hello!

Did you try the link above?
You just need to unzip the package and execute the MonaTiny.exe file.

Let us know if you need more information
